README.md

```diff
@@ -28,50 +28,50 @@ This repo aggregates all audits performed on Sablier smart contracts by third-pa
-| Rahul Saxena   | Solo  | [`8bd57eb`][8bd57eb]                       | [2023-04-24](./lockup/v1.0/20230424_rahul.pdf)                                                              |
-| Iaroslav Mazur | Solo  | [`8bd57eb`][8bd57eb]                       | [2023-04-04](./lockup/v1.0/20230404_iaro.pdf)                                                               |
-| HYDN           | Firm  | [`8bd57eb`][8bd57eb]                       | [2023-04-03](./lockup/v1.0/20230403_hydn.pdf)                                                               |
-| Hake           | Solo  | [`8bd57eb`][8bd57eb]                       | [2023-03-31](./lockup/v1.0/20230331_hake.pdf)                                                               |
+| Auditors       | Types | Commits                                    | Reports                                                                                                         |
+| :------------- | :---- | :----------------------------------------- | :-------------------------------------------------------------------------------------------------------------- |
+| Iaroslav Mazur | Solo  | [`5997ac0`][5997ac0], [`005df5f`][005df5f] | [2023-06-13](./lockup/v1.0.0/20230613_iaro_core.pdf), [2023-07-09](./lockup/v1.0.0/20230709_iaro_periphery.pdf) |
+| Cantina        | Firm  | [`8bd57eb`][8bd57eb]                       | [2023-06-08](./lockup/v1.0.0/20230608_cantina.pdf)                                                              |
+| Rahul Saxena   | Solo  | [`8bd57eb`][8bd57eb]                       | [2023-04-24](./lockup/v1.0.0/20230424_rahul.pdf)                                                                |
+| Iaroslav Mazur | Solo  | [`8bd57eb`][8bd57eb]                       | [2023-04-04](./lockup/v1.0.0/20230404_iaro.pdf)                                                                 |
+| HYDN           | Firm  | [`8bd57eb`][8bd57eb]                       | [2023-04-03](./lockup/v1.0.0/20230403_hydn.pdf)                                                                 |
+| Hake           | Solo  | [`8bd57eb`][8bd57eb]                       | [2023-03-31](./lockup/v1.0.0/20230331_hake.pdf)                                                                 |
 
-### v1 (Legacy)
+### Legacy
 
 [fc54b02]: https://github.com/sablier-labs/v1-protocol/tree/fc54b0233e186232f6d724fa89d1cf7c1f45c688
 
-| Auditors   | Types | Commits              | Reports                                                  |
-| :--------- | :---- | :------------------- | :------------------------------------------------------- |
-| Quantstamp | Firm  | [`fc54b02`][fc54b02] | [2019-11-25](./lockup/v1-legacy/20191125_quantstamp.pdf) |
+| Auditors   | Types | Commits              | Reports                                               |
+| :--------- | :---- | :------------------- | :---------------------------------------------------- |
+| Quantstamp | Firm  | [`fc54b02`][fc54b02] | [2019-11-25](./lockup/legacy/20191125_quantstamp.pdf) |
```

Comment:
it should be `./legacy/20191125_quantstamp.pdf`.

Reply:
correct